liquid/stream.py

```diff
@@ -94,7 +94,7 @@ def next(self):
 			str: the next character
 		"""
 		ret = self.stream.read(1)
-		self.cursor += len(ret.encode('utf-8'))
+		self.cursor = self.stream.tell()
```

Comment:
I realized that unicode will affect all cursor operations in this class (i.e. in the next function `back`). Maybe we just put it as a property and let the stream itself to tell where the cursor is?

```
@property
def cursor(self):
  return self.stream.cursor
```

Reply:
That makes sense.  I'll update this PR